worldedit-core/src/main/java/com/sk89q/worldedit/function/operation/ForwardExtentCopy.java

```diff
@@ -427,8 +428,8 @@ public void cancel() {
         return ImmutableList.of(
                 TranslatableComponent.of("worldedit.operation.affected.block",
                         TextComponent.of(affectedBlocks)).color(TextColor.LIGHT_PURPLE),
-                TranslatableComponent.of("worldedit.operation.affected.biome",
-                        TextComponent.of(affectedBiomeCols)).color(TextColor.LIGHT_PURPLE),
+                //TranslatableComponent.of("worldedit.operation.affected.biome",
```

Comment:
There seems to be no nice way to count affected biomes atm, so I commented it out